src/index.html

```diff
@@ -30,7 +30,7 @@
 
     <!-- for Facebook -->
     <meta property="og:title" content="CARE's World of Impact" />
-    <meta property="og:image" content="" />
+    <meta property="og:image" content="http://care-usa.herokuapp.com/care-map.jpg" />
```

Comment:
I'm not sure. The thing is that I asume that route will be used by FB. And if we give something lie `/my-image.jpg`, it will understand that the image is at it server. 

But if you want, we can try.